computer-use-server/app.py

```
@@ -437,6 +441,15 @@ async def upload_file(chat_id: str, filename: str, file: UploadFile = File(...))
         # Calculate MD5 for confirmation
         md5_hash = hashlib.md5(content).hexdigest()
 
+        # Tier 6 — refresh MCP resources for this chat so the new file
+        # appears in resources/list without reconnecting. Swallow errors
+        # so the upload itself succeeds even if MCP resource sync fails.
+        try:
+            from mcp_resources import sync_chat_resources
+            await sync_chat_resources(chat_id)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)